### PR TITLE
Update Governance Files

### PR DESCRIPTION
**Pull request by tanto259**
Signed-off-by: Hartanto Ario Widjaya <tanto259@users.noreply.github.com>

## Proposed changes

This pull request is intended to update and add some governance files in line with the revision to the Open Mainframe Project's Project Lifecycle.

Mainly sourced from the project-template repository, but modified for this course. This includes necessary modification to achieve a gold CII Best Practices badge.

GitHub Issue and PR template is also added per the project-template but modified for the purpose of this course.

Additionally, some typos are fixed with this PR.

No testing are required since this is a documentation-change only.

## Type of change

What type of changes does your PR introduce to the COBOL Programming Course? _Put an `x` in the boxes that apply._

- [ ] Bug fix (change which fixes one or more issues)
- [ ] New feature (change which adds functionality or feature to the course)
- [ ] Translations (change which adds or modify translations of the course)
- [x] Documentation (change which modify documentation related to the course)
- [ ] This change requires an update to the course's z/OS environment

## Checklist:

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [Contributing Guideline](https://github.com/openmainframeproject/cobol-programming-course/blob/master/CONTRIBUTING.md)
- [x] I have included a title and description for this PR
- [x] I have DCO-signed all of my commits that are included in this PR
- [ ] I have tested it manually and there are no regressions found
- [ ] I have commented my code, particularly in hard-to-understand areas (if appropriate)
- [x] I have made corresponding changes to the documentation (if appropriate)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+name: Bug Report
+description: Create a report to help us improve the COBOL Programming Course
+labels: ['bug']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting a bug report. Please fill up the information below and we will get back to you as soon as possible.
+
+        Before opening an issue, please search our existing issue: https://github.com/openmainframeproject/cobol-programming-course/issues.
```
Comment (MikeBauerCA) [inline .github/ISSUE_TEMPLATE/bug_report.yml:10]: please search our existing issues...

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+name: Feature Request
+description: Suggest an idea for the COBOL Programming Course
+labels: ['enhancement']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting a feature request. Please fill up the information below and we will get back to you as soon as possible.
```
Comment (MikeBauerCA) [inline .github/ISSUE_TEMPLATE/feature_request.yml:8]: Please fill in the information...

Also, do we want to recommend searching the issues to see if the enhancement exists like you did for the Bug Report template?

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context to communicate to the maintainers why we should accept this pull request.
+
+Fixes # (issue)
+
+## Type of change
+
+What type of changes does your PR introduce to the COBOL Programming Course? _Put an `x` in the boxes that apply._
+
+- [ ] Bug fix (change which fixes one or more issues)
+- [ ] New feature (change which adds functionality or feature to the course)
```
Comment (MikeBauerCA) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:12]: adds functionality or features

File: ROADMAP.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Roadmap
+
+These roadmap list out the possible and planned future contents for the COBOL Programming Course. If you are interested in contributing content, please review our [contribution guidelines](https://github.com/openmainframeproject/cobol-programming-course/blob/master/CONTRIBUTING.md).
```
Comment (MikeBauerCA) [inline ROADMAP.md:3]: This roadmap lists out...

**Review by MikeBauerCA — COMMENTED** [review]
Overall, this is an awesome PR @tanto259! Very well done 👍 . I made a few small suggestions. Just let me know when you have finished evaluating my comments and I will move this PR to approved and merge it. Thank you!

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 2 above)
Comment (tanto259) [reply .github/ISSUE_TEMPLATE/feature_request.yml]: Updated to include the recommendation

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Security Policy
+
+## Considerations
+
+The COBOL Programming Course does not produce any kind of software. This project produces a technical course aimed to educate people on COBOL programming. The output of this project came in the form of PDF documents detailing the course content. A Markdown version is also available on the repository.
+
+## Supported Version
+
+Only the [latest version](https://github.com/openmainframeproject/cobol-programming-course/releases/latest) of the course is supported with security updates.
```
[the remark this answers is not in the document]
Comment (tanto259) [reply SECURITY.md]: Seems redundant... removed in the latest commit

**Comment by tanto259** [comment]
PR updated. Thanks, @MikeBauerCA!

**Review by MikeBauerCA — APPROVED** [review]

File: ROADMAP.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# Roadmap
+
+These roadmap lists out the possible and planned future contents for the COBOL Programming Course. If you are interested in contributing content, please review our [contribution guidelines](https://github.com/openmainframeproject/cobol-programming-course/blob/master/CONTRIBUTING.md).
```
Comment (MikeBauerCA) [inline ROADMAP.md:3]: Please change "These roadmap" to "This roadmap"

**Pull request merged**